### PR TITLE
Add JUnit tests for StringUtil#splitByWhiteSpace

### LINKED ISSUE #251: Add JUnit test for splitByWhitespace in StringUtilTest
The `splitByWhitespace` method, located in `StringUtil.java`, was not tested in `StringUtilTest.java`, please help to add in the test. Thanks!

### PR DESCRIPTION
**Pull request by lulucopter**
Resolves #251

Add test cases for splitByWhiteSpace method
Also fixes a bug found while tests were conducted for method.

### CODE REVIEW

File: src/test/java/seedu/address/commons/util/StringUtilTest.java [hunk 1]
```diff
@@ -251,4 +254,19 @@ public void trimAndReplaceMultipleSpaces() {
...
 
+    //---------------- Tests for splitByWhiteSpace --------------------------------------
+    @Test
+    public void splitByWhiteSpaceTest() {
+        assertEquals(StringUtil.splitByWhitespace(null), null); // null string
+        assertArrayEquals(StringUtil.splitByWhitespace(""), new String[]{""}); //empty string
+        assertArrayEquals(StringUtil.splitByWhitespace("test"), new String[]{"test"}); // no space
+        assertArrayEquals(StringUtil.splitByWhitespace("test "), new String[]{"test"}); // 1 trailing space
+        assertArrayEquals(StringUtil.splitByWhitespace(" test "), new String[]{"test"}); // 1 Leading and trailing space
+        assertArrayEquals(StringUtil.splitByWhitespace(" \t \n test  \t \n "), new String[]{"test"}); // Multiple Leading and trailing space
```
Comment (maryjess) [inline src/test/java/seedu/address/commons/util/StringUtilTest.java:265]: Please fix the checkstyle issues first!

**Review by maryjess — APPROVED** [review]
LGTM!

**Review by deeyonn — APPROVED** [review]
LGTM

**Comment by deeyonn** [comment]
Somehow the merge made some changes that removed a essential curly brace. I have fixed it in my PR #269

**Comment by cliftonfelix** [comment]
> Somehow the merge made some changes that removed a essential curly brace. I have fixed it in my PR #269

My bad, should be when I was resolving the conflicts through Github directly

**Pull request merged**